node/resolvers/httpResolver.ts

```diff
@@ -3,15 +3,15 @@ import { ColossusContext, IOContext } from 'colossus'
 import { prop } from 'ramda'
 import * as parse from 'url-parse'
 
-const defaultMerge = (bodyData, resData) => resData
+const defaultMerge = (bodyData, resData, response) => resData
 
 export type URLBuilder = (account: string, data: any, root: any) => string
 
 export type DataBuilder = (data: any) => any
 
 export type HeadersBuider = (ioContext: IOContext) => Record<string, string>
 
-export type ResponseMerger = (bodyData: any, responseData: any) => any
+export type ResponseMerger = (bodyData: any, responseData: any, response?: any) => any
```

Comment:
why you don´t just pass the whole response on the responseData field, and inside the function, you check if it's a response or not?

Reply:
Because you will need to check is the response is null or not anyway...